landbosse/model/SitePreparationCost.py

```diff
@@ -565,11 +727,19 @@ def outputs_for_detailed_tab(self, input_dict, output_dict):
             'value': self.output_dict['crane_path_width_m']     #TODO: Rename variable to: crane_path_width_ft
         })
 
+        if not input_dict['road_distributed_wind']:
+            result.append({
+                'unit': 'm',
+                'type': 'variable',
+                'variable_df_key_col_name': 'Road length',
+                'value': float(self.output_dict['road_length_m'])
+            })
+
         result.append({
-        'unit': 'm',
-        'type': 'variable',
-        'variable_df_key_col_name': 'Road length',
-        'value': float(self.output_dict['road_length_m'])
+            'unit': 'm',
+            'type': 'variable',
+            'variable_df_key_col_name': 'Road width',
+            'value': self.output_dict['road_width_m']
         })
```

Comment:
Why was this changed? Does distributed wind not report any road length? Also, it looks like road_width might be duplicated now... Please check.

Reply:
@eberlea given the highly varying nature of road lengths in DW projects (including projects with no roads at all), the team couldn't come up with a valid relationship for road length. It made most sense instead, to add road length as a user input (for DW BOS modeling). So basically, in the project_list spreadsheet, the column 'Road Adder (m)' is used as the user defined input for total road length in DW projects. 

Good catch on the double reporting of ['road_width_m']. This has now been fixed.